README.md

```diff
@@ -136,6 +150,7 @@ The following sections provide a full list of configuration in- and output varia
 | spot\_price | The spot price ceiling for spot instances | `string` | `"0.1"` | no |
 | subdomain | The subdomain to be added to the apex domain. If subdomain is set, it will be appended to the apex domain in  `jx-requirements-eks.yml` file | `string` | `""` | no |
 | tls\_email | The email to register the LetsEncrypt certificate with. Added to the `jx-requirements.yml` file | `string` | `""` | no |
+| vault\_url | URL to an external Vault instance in case Jenkins X shall not create its own system Vault | `string` | `""` | no |
```

Comment:
```suggestion
| vault\_url | URL to an external Vault instance in case Jenkins X does not create its own system Vault | `string` | `""` | no |
```

Reply:
Fixed. FYI @jha-cloudbees, these tables for the lists of in- and outputs are generated from  `variables.tf` and `outputs.tf`. So if we need to change something in this table, it is important that we change the underlying description in the source files, otherwise, it will keep reappearing.